internal/action/main_test.go

```diff
@@ -0,0 +1,291 @@
+package action
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-tfe"
+	"github.com/stretchr/testify/assert"
+)
+
+var workspacePrefix string = "action-test"
+
+func NewTestRunConfig() *RunConfig {
+	return &RunConfig{
+		Token:                  os.Getenv("tf_token"),
+		Organization:           os.Getenv("tf_organization"),
+		Host:                   "app.terraform.io",
+		Name:                   fmt.Sprintf("%s-%d", workspacePrefix, time.Now().Unix()),
+		Import:                 true,
+		Apply:                  true,
+		TFEProviderVersion:     "0.25.3",
+		RunnerTerraformVersion: "1.0.5",
+		TerraformVersion:       "1.0.5",
+	}
+}
+
+func RemoveTestWorkspaces(ctx context.Context, client *tfe.Client, organization string, prefix string) error {
+	workspaces, err := client.Workspaces.List(ctx, organization, tfe.WorkspaceListOptions{
+		Search: &prefix,
+		ListOptions: tfe.ListOptions{
+			PageSize: 100,
```

Comment:
This is used in a bunch of places, not sure why mnd isn't catching, the existing `maxPageSize` constant should be used.